spec/guess-the-number-spec.js

```diff
@@ -0,0 +1,23 @@
+const chai = require( 'chai' )
+const { expect } = require( 'chai' )
+const chaiChange = require ( 'chai-change' )
+const {
+  guessTheNumber,
+  randomNumberGenerator } = require ( '../src/guess-the-number' )
+
+describe( 'guessTheNumber', () => {
+  'use strict'
+
+  it( 'exists', () => {
+    expect( guessTheNumber ).to.be.a( 'function' )
+  })
+
+  context( 'randomNumberGenerator()', () =>
+    it( 'generates a random number n', () => {
+      expect( randomNumberGenerator() ).to.be.a( 'number' )
+    }),
+    it( 'generates a number whose value is between 0 and 100', () => {
```

Comment:
I couldn't think of a way to check for randomness, so I opted to make sure that the number will always be in the appropriate range. 

http://chaijs.com/api/bdd/#method_within

Am I misunderstanding the proper use of the within assertion? It "asserts that the target is within a range."